posthog/temporal/ai/session_summary/summarize_session.py

```diff
@@ -384,6 +393,115 @@ async def run(self, inputs: SingleSessionSummaryInputs) -> None:
             )
 
 
+async def ensure_llm_single_session_summary(inputs: SingleSessionSummaryInputs):
+    if inputs.video_validation_enabled == "full":
```

Comment:
Nitpick: No need to keep the code nested. If not full - start and return regular activity, no need for else afterward.